### PR TITLE
DON-950: Migrate to use new Angular template control flow features

### PR DESCRIPTION
**Pull request by bdsl**
Changes generated by running:
> ng generate @angular/core:control-flow

I suggest turning off whitestpace display when reviewing. As changes are autogenerated probably reviewing a sample is enough - may not be necassary to review every single change.

### CODE REVIEW

File: src/app/campaign-details/campaign-details.component.html [hunk 1]
```diff
@@ -177,91 +192,106 @@
...
-              <li class="c-budget__item">
-                <span class="c-budget__value b-semi-bold">{{ detail.amount | currency : campaign.currencyCode : 'symbol' : currencyPipeDigitsInfo }}</span>
-                &ndash; {{ detail.description }}
-              </li>
-            </ul>
-
-            <div class="c-alternative-funds" *ngIf="campaign.alternativeFundUse">
+          @if (campaign.updates.length > 0) {
+            <mat-tab label="Updates">
+              @for (update of campaign.updates; track update) {
```
Comment (bdsl) [inline src/app/campaign-details/campaign-details.component.html:225]: https://angular.io/guide/control_flow#track-for-calculating-difference-of-two-collections

File: src/app/donation-thanks/donation-thanks.component.html [hunk 2]
```diff
@@ -1,211 +1,231 @@
...
+    <div id="banner-thank-you"></div>
+  }
+
+  @if (!complete && !noAccess && !timedOut) {
+    <mat-spinner class="complete-spinner" color="primary"
+    diameter="30" aria-label="Loading donation details"></mat-spinner>
+  }
+
+  @if (isDataLoaded) {
+    <div>
```
Comment (bdsl) [inline src/app/donation-thanks/donation-thanks.component.html:12]: this div is probably redundant, would be nice if the script could have removed it and similar, but I don't think it's a big issue.

**Comment by bdsl** [comment]
I noticed I merged this into develop by mistake along with something else. Now reverted.

**Comment by dorota-joanna** [comment]
LGTM!

**Pull request merged**